eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -613,7 +613,7 @@ object Peer {
     def channels: Map[_ <: ChannelId, ActorRef] // will be overridden by Map[FinalChannelId, ActorRef] or Map[ChannelId, ActorRef]
   }
   case class Nothing() extends Data { override def address_opt = None; override def channels = Map.empty }
-  case class DisconnectedData(address_opt: Option[InetSocketAddress], channels: Map[FinalChannelId, ActorRef], nextReconnectionDelay: FiniteDuration = 10 seconds) extends Data
+  case class DisconnectedData(address_opt: Option[InetSocketAddress], channels: Map[FinalChannelId, ActorRef], nextReconnectionDelay: FiniteDuration = randomizeDelay(10 seconds)) extends Data
```

Comment:
How about completely removing the default value for `nextReconnectionDelay`? Default is used in 3 places and would be more explicit to set the new reconnection delay when creating the `DisconnectedData` in each instance.

Reply:
I'm not a fan of default values in general, but here I think it's pretty elegant: it allows us to use the same initial value without repeating ourselves.